packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -504,11 +499,13 @@ class Treemap extends StatelessWidget {
           height: constraints.maxHeight,
         );
         if (levelsVisible <= 1) {
+          // If this is the second to the last level, paint all widgets in the last level.
```

Comment:
nit: add reasoning why (e.g. to improve performance)

Reply:
we want someone in the future who is looking at this code to understand why we are painting some cells and using widgets for others

Reply:
Sounds good. done